server/mod.ts

```diff
@@ -24,6 +24,7 @@ export type ServerOptions = Omit<ServeInit, "onError"> & {
   fetch?: FetchHandler;
   ssr?: SSR;
   onError?: ErrorCallback;
+  port?: number;
```

Comment:
instead, can you please remove the `port` option and add `Partial<Omit<ServeInit, "onError">>` ?